AGGREGATE.md

```diff
@@ -108,14 +102,12 @@ which generates aggregation keys.
 [
 // Each dict independently adds pieces to multiple source keys.
 {
-  "key_piece": "10",// Conversion type purchase = 2
-  "key_offset": 9,
+  "key_piece": "0x400",// Conversion type purchase = 2 at a 9 bit offset
   // Apply this suffix to:
   "source_keys": ["campaignCounts"]
 },
 {
-  "key_piece": "10101",// Purchase category shirts = 21
-  "key_offset": 7,
+  "key_piece": "0xA80",// Purchase category shirts = 21 at a 7 bit offset
```

Comment:
nit: do you think something like `= 21 (0b10101)` to spell out the binary might be useful? This might help connect the idea of a bit offset and the resulting contributions at the bottom.

Reply:
It would be lovely to be able to support bit shift operations: 21 << 7 :)
(not realistic)

Reply:
> It would be lovely to be able to support bit shift operations: 21 << 7 :)
> (not realistic)

It is realistic, it is just spelled "offset" in the existing documentation :) I can update comment here to spell out more clearly how these were generated.